website/src/Website.vue

```diff
@@ -19,6 +19,8 @@
 
 			</div>
 
+			<p>Feel free to contact us or report bugs at holisticweb@gmail.com</p>
```

Comment:
Let's either use an email address that we have or set this email up before deploying this. We should also add a reference to our email to the main Readme so we remember to check it sometimes.

Reply:
I made: holisticweb.contact@gmail.com
I'll send you the password.